spec/models/plant_spec.rb

```diff
@@ -8,4 +8,45 @@
   describe 'associations' do
     it { should belong_to(:garden) }
   end
+  describe 'class methods' do
```

Comment:
Def a wave moving forward. I'm a bit too lazy to go back and refactor everything but this is sweet 🎂